extensions/windows-event-log/ConsumeWindowsEventLog.h

```diff
@@ -167,23 +159,21 @@ class ConsumeWindowsEventLog : public core::Processor {
   uint64_t batch_commit_size_{};
   bool cache_sid_lookups_ = true;
 
-  SMART_ENUM(JSONType,
-      (None, "None"),
-      (Raw, "Raw"),
-      (Simple, "Simple"),
-      (Flattened, "Flattened"))
-
-  struct OutputFormat {
-    bool xml{false};
-    bool plaintext{false};
-    struct JSON {
-      JSONType type{JSONType::None};
-
-      explicit operator bool() const noexcept {
-        return type != JSONType::None;
-      }
-    } json;
-  } output_;
+  SMART_ENUM(OutputFormatT,
+    (XML, "XML"),
+    (BOTH, "Both"),
+    (PLAINTEXT, "Plaintext"),
+    (JSON, "JSON")
+  );
+
+  SMART_ENUM(JsonFormatT,
+    (RAW, "Raw"),
+    (SIMPLE, "Simple"),
+    (FLATTENED, "Flattened")
+  );
```

Comment:
the linter doesn't like the `;` after the smart enums

Reply:
removed